tests/unit/httpsTests.js

```diff
@@ -47,27 +50,26 @@ const testClients = [
 ];
 
 function extractPost(req, cb) {
-    let body = '';
-    req.on('data', data => body += data).on('error', cb).on('end', () => {
-        try {
-            body = JSON.parse(body);
-        } catch (e) {
-            return cb(e);
-        }
-        return cb(null, body);
+    const body = [];
+    req.on('data', data => body.push(data)).on('error', cb).on('end', () => {
+        cb(null, querystring.parse(body.join('')));
     });
 }
 
 function handler(req, res) {
     if (req.method === 'POST') {
         extractPost(req, (err, body) => {
             if (err || (body && (body.name !== accountName
-                        || body.emailAddress !== accountOptions.email
-                        || body.saltedPwd !== accountOptions.password))) {
+                        || body.emailAddress !== accountOptions.email))) {
+                let code = 'Unknow';
```

Comment:
It's hard to follow what's going here, can you add a comment explaining it?
Also, `Unknow` => `Unknown`